projects/kit/src/lib/masks/number/utils/stringify-number-without-exp.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Convert number to string with replacing exponent part on decimals
+ *
+ * @param value the number
+ * @return string representation of a number
+ */
+export function stringifyNumberWithoutExp(value: number): string {
```

Comment:
Utility is copied from from Taiga UI.
https://github.com/Tinkoff/taiga-ui/blob/main/projects/core/utils/format/number-to-string-without-exp.ts#L7